src/go/k8s/controllers/test/cluster_controller_test.go

```diff
@@ -7,13 +7,16 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0
 
-package redpanda_test
+// nolint:testpackage // this name is ok
+package test
 
 import (
 	"context"
 	"fmt"
 	"time"
 
+	"github.com/redpanda-data/redpanda/src/go/k8s/controllers/redpanda"
+
```

Comment:
NIT: Why this import was moved up? I think you always mention that order should be:
1. stdlib
2. external
3. internal